internal/util/exit.go

```diff
@@ -1,5 +1,29 @@
 package util
 
-import "os"
+import (
+	"os"
 
-var Exit = os.Exit
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var (
+	ShouldExit   = true
```

Comment:
Make the name something that makes it super obvious that normal people should not be using this.
I usually add "for testing" to then end of the name (with matching casing/style).
For example:
https://github.com/edulinq/autograder-server/blob/main/internal/util/dir.go#L15